css-transitions-2/Overview.bs

```diff
@@ -72,12 +72,29 @@ spec:web-animations-1; type:dfn;
 
 ## The 'transition-property' Property ## {#transition-name-property}
 
+ISSUE: Insert text from [[css-transitions-1#transition-property-property]]
+and backport the following paragraph.
+
 Although the 'transition-property' may specify shorthand properties and the
 ''all'' keyword, individual transitions are generated for each longhand
 sub-property that is animatable. The <dfn>expanded transition property
 name</dfn> of a transition is the name of the longhand sub-property for which
 the transition was generated (e.g. 'border-left-width').
 
+ISSUE: Update the defining instance of [=transitionable=]
+once it is ported from Level 1.
+
+In CSS Transitions Level 2,
+property values are [=transitionable=]
+<em>unless</em> they have an [=animation type=] that is [=not animatable=],
+i.e. [=discrete=] values <em>are</em> [=transitionable=],
```

Comment:
Maybe make this its own sentence and make it a note?

Reply:
(OK, github reviews doesn't actually associate my comment with the text I selected, so I was referring to the bit starting with "i.e.".)

Reply:
Rewrote it differently, but kept it normative because the bit about transitioning at 50% seemed worth stating explicitly.